src/main/java/lesson02/tasksresolve/RangePrinter.java

```diff
@@ -0,0 +1,42 @@
+package lesson02.tasksresolve;
+/*
+Task 1 (RangePrinter)
+Вывести на экран целые числа от first до last.
+Ограничения: нет.
+void printRange(int first, int last)
+Входные аргументы - int first, int last
+Тип результата - Вывод в консоль
+ */
+
+import lesson02.utility.ConsoleInput;
+
+public class RangePrinter {
+    private static final String MESSAGE_FOR_FIRST_NUMBER_INPUT = "Please input first number: ";
+    private static final String MESSAGE_FOR_LAST_NUMBER_INPUT = "Please input last number: ";
+
+    public static void main (String[] args)  {
+        try {
+            printRange(
+                    ConsoleInput.inputNumberFromConsole(MESSAGE_FOR_FIRST_NUMBER_INPUT),
+                    ConsoleInput.inputNumberFromConsole(MESSAGE_FOR_LAST_NUMBER_INPUT)
+            );
+        } catch (Exception e) {
+            System.exit(-1);
```

Comment:
More useful print some message in case exception or print stack trace.

Reply:
Fixed